### PR TITLE
PHP8 upgrade

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: versions.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+behat/behat               v3.11.0            Scenario-oriented BDD framework for PHP
```
Comment (Schparky) [inline versions.txt:1]: I wonder why `behat/behat` is listed. Seems it should exclude dev dependencies.

**Review by Schparky — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -5,4 +5,4 @@ install:
 	docker-compose run --rm test composer install
 
 update:
-	docker-compose run --rm test composer update
+	docker-compose run --rm test php -version && composer update && composer show -D > versions.txt
```
Comment (forevermatt) [inline Makefile:8]: I don't think the `&&` works unless you use `bash -c "stuff"`:
```suggestion
	docker-compose run --rm test bash -c "php -version && composer update && composer show -D > versions.txt"
```

**Review by forevermatt — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (Baggerone) [reply Makefile]: I went ahead and made the change, but it seems to run the same on my machine either way.

File: versions.txt (hunk 1 above)
Comment (forevermatt) [reply versions.txt:1]: It looks like `composer show` has a `--no-dev` flag for when that's the desired behavior.

**Review by forevermatt — APPROVED** [review]

**Pull request merged**